cumulus/parachains/runtimes/testing/penpal/src/lib.rs

```diff
@@ -230,7 +230,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("penpal-parachain"),
 	impl_name: create_runtime_str!("penpal-parachain"),
 	authoring_version: 1,
-	spec_version: 10000,
+	spec_version: 1,
```

Comment:
and here?

Reply:
This is a test runtime, that should never require a spec version bump.